src/operations/indexes.ts

```diff
@@ -360,7 +360,12 @@ export class DropIndexOperation extends CommandOperation<Document> {
 }
 
 /** @public */
-export type ListIndexesOptions = AbstractCursorOptions;
+export type ListIndexesOptions = AbstractCursorOptions & {
+  /** @internal TODO(NODE-5688): make this public */
+  timeoutMode?: CursorTimeoutMode;
+  /** @internal TODO(NODE-5688): make this public */
+  omitMaxTimeMS?: boolean;
```

Comment:
Double check, we don't want this public, we may need to add it here though b/c of passing this to the cursor.